modules/retail/pages/retail-uyuni-server-setup.adoc

```diff
@@ -0,0 +1,233 @@
+[[retail-server-setup]]
+= Retail Uyuni Server Setup
+
+This section covers {productname} Server setup, using these procedures:
+
+* Start {productname} setup with {yast}
+* Create the main administration account with the {productname} {webui}
+* Name your base organization and add login credentials
+* Synchronize the openSUSE Leap channel
++
+// ^^^ CHECKIT
+
+
+
+[[retail-server-setup-yast]]
+== Set up {productname} with {yast}
+
+This section will guide you through {productname} setup procedures.
+
+.Procedure: {productname} Setup
+. Log in to the {productname} Server and start {yast}.
+
+. In {yast}, navigate to menu:Network Services[Uyuni Setup] to begin the setup.
+
+. From the introduction screen select menu:Uyuni Setup[Set up Uyuni from scratch] and click btn:[Next] to continue.
+
+. Enter an email address to receive status notifications and click btn:[Next] to continue.
+{productname} can sometimes send a large volume of notification emails.
+You can disable email notifications in the {webui} after setup, if you need to.
+
+. Enter your certificate information and a password.
+Passwords must be at at least seven characters in length, and must not contain spaces, single or double quotation marks (``'`` or ``"``), exclamation marks (``!``), or dollar signs (``$``).
+Always store your passwords in a secure location.
++
+
+[IMPORTANT]
+====
+You must have the certificate password to set up a {productname} Proxy Server.
+====
+
+. Click btn:[Next] to continue.
+
+. From the menu:Uyuni Setup[Database Settings] screen, enter a database user and password and click btn:[Next] to continue.
+Passwords must be at at least seven characters in length, and must not contain spaces, single or double quotation marks (``'`` or ``"``), exclamation marks (``!``), or dollar signs (``$``).
+Always store your passwords in a secure location.
+
+. Click btn:[Next] to continue.
```

Comment:
You just said this in the previous step, I think this one can be deleted.

Reply:
no, this one is the database password, the previous one was for the cert.